src/main/java/org/wso2/carbon/esb/connector/MQReportConstants.java

```diff
@@ -0,0 +1,40 @@
+/*
+*  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+package org.wso2.carbon.esb.connector;
+
+/**
+ * Constants for report message
+ */
+public class MQReportConstants {
+    public static final String MQFB_EXPIRATION = "Feedback indicating that the message was discarded because it had not been removed from the destination queue before its expiry time had elapsed";
```

Comment:
Since we point a separate reply queue, handle this separately, we do not need to decode this message codes.